parachain/pallets/inbound-queue/src/test.rs

```diff
@@ -111,6 +142,8 @@ impl inbound_queue::Config for Test {
 	type XcmSender = ();
```

Comment:
Maybe you can use a `mockXcmSender` here just need to implement the `SendXcm` trait return some mock value [here](https://github.com/paritytech/polkadot/blob/ff869d0cda604f9adf2b6f886a813552b6fb848f/xcm/src/v3/traits.rs#L512-L533)